R/encoloc.R

```diff
@@ -99,11 +99,32 @@ calculate_cumsum <- function(coloc_results) {
 load_and_extract_ld_matrix <- function(ld_meta_file_path, analysis_region, variants) {
   # This is a placeholder for loading LD matrix, adjust as per your actual function
   ld_ref <- load_LD_matrix(LD_meta_file_path = ld_meta_file_path, region = analysis_region)
-  ext_ld <- ld_ref[[2]][variants, variants]
+  ext_ld <- ld_ref$combined_LD_matrix[variants, variants]
   ext_ld
 }
 
-#' @importFrom susieR get_purity
+# Subsample and compute min, mean, median and max abs corr.
```

Comment:
Sorry you can still use susieR::get_purity ... it is just that this susieR is from gaow/susieR which is. the case in the pecotmr image already. There is no need to redefine it. 

At some point of development I'm going to merge gaow/susieR to stephenslab/susieR.